### PR TITLE
Fix #9455 Longitudinal profile fixes

### LINKED ISSUE #9455: [Enhancements] - longitudinal profile improvements
## Description

There are a few issues that need to be handled

- [x] There is a problem with the display of the Settings button, which is "hidden" and inaccessible in the following configuration: Lenovo T590 laptop, Chrome browser, full screen window, 100% zoom. The problem can be circumvented by, for example, setting the browser zoom to 90%, but it is not possible to offer this workaround

Image

- [x] The focus of the graph is not done automatically, which makes it unreadable most of the time (because of the outliers in the "BDBATHY_CNR" layer taken into account the values outside the Rhône which are by default at 999999 meters)

Image

Is it possible to do something so that values that are improbable are not taken into account?

- [x] The client provided an updated translation files for FR to contribute with some language fixes (
[data.fr-FR-version_Atolcd.zip](https://github.com/geosolutions-it/MapStore2/files/12598502/data.fr-FR-version_Atolcd.zip))

- [x] Not browser related

<details><summary> <b>Browser info</b> </summary>

(use this site: <a href="https://www.whatsmybrowser.org/">https://www.whatsmybrowser.org/</a> for non expert users)

| Browser Affected | Version |
|---|---|
|Internet Explorer| |
|Edge| |
|Chrome| |
|Firefox| |
|Safari| |
</details>

## Other useful information

https://github.com/geosolutions-it/MapStore-C265/issues/19

### PR DESCRIPTION
**Pull request by MV88**
* fr locales fixed
* no data threshold applied
* dropup longitudinal menu

## Description

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

fix #9455

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/plugins/LongitudinalProfileTool.jsx [hunk 1]
```diff
@@ -70,6 +70,7 @@ import { createPlugin } from '../utils/PluginsUtils';
  * @prop {Object} cfg.config the plugin configuration
  * @prop {string} cfg.config.wpsurl optional, the geoserver url the the wps endpoint to use. It can be an absolute url. default is "/geoserver/wps".
  * @prop {string} cfg.config.chartTitle the default title of the chart
+ * @prop {number} cfg.config.noDataThreshold the number that beyond it will exclude data
```
Comment (offtherailz) [inline web/client/plugins/LongitudinalProfileTool.jsx:73]: It is ok to make it not configurable by UI? It depends on the layer, maybe it should be configured for layer or manually inserted.

**Review by offtherailz — CHANGES_REQUESTED** [review]
- [ ] This is not very good looking. You should make it look like a menu item. 
Notice that you will receive `menuItem=true` in props when the Sidebar item is rendered as an extra-item.

image


- [ ] Is it possible to test the threshold with our data? I think it is. I found this issue that is very similar with the one provided by the client. Can you find the value for this? 
image

File: web/client/plugins/LongitudinalProfileTool.jsx (hunk 1 above)
Comment (MV88) [reply web/client/plugins/LongitudinalProfileTool.jsx:73]: it was requested to develop the quickest solution

**Comment by MV88** [comment]
>  Is it possible to test the threshold with our data? I think it is. 

yes you can configure a dedicated noDataThreshold value in the context in the related cfg plugin

>  This is not very good looking. You should make it look like a menu item.
Notice that you will receive menuItem=true in props when the Sidebar item is rendered as an extra-item.

sure, will check

**Review by offtherailz — APPROVED** [review]

File: web/client/plugins/LongitudinalProfileTool.jsx (hunk 1 above)
Comment (offtherailz) [reply web/client/plugins/LongitudinalProfileTool.jsx:73]: Ok. If @tdipisa is aware of this. it is ok for me.

**Comment by ElenaGallo** [comment]
Test passed, @MV88 please backport this to stable branch. Thanks

**Pull request merged**